federator/src/contracts/FederationFactory.js

```diff
@@ -26,7 +26,7 @@ module.exports = class FederationFactory extends ContractFactory {
         let federationContract = this.getContractByAbi(abiFederationNew, address, web3);
         const version = await this.getVersion(federationContract);
 
-        if (version === 'v2') {
+        if (version === 'v2' || version === 'v3') {
```

Comment:
We want to move to version 3 and leave version 2 as the old one like this:
```if (version === 'v3') {
          return new FederationInterfaceV3(this.config, federationContract);
        } else if (version === 'v2') {
         return new FederationInterfaceV2(this.config, federationContract);```
       
     We should tackle this when we restructure the deploy of the old Contracts

Reply:
> ```
>      We should tackle this when we restructure the deploy of the old Contracts
> ```

agree